### PR TITLE
Storage of data locally

### PR DESCRIPTION
**Pull request by MenendezON**
This pull request concerns the storage of data locally. It stores all values from the form and once the page is reloaded, it prefills the form from the data storage.

### CODE REVIEW

File: public/script.js [hunk 1]
```diff
@@ -78,4 +55,22 @@ function validateForm() {
...
+
+// Activity 4 : Local storage
+const namef = document.getElementById('name');
+const email = document.getElementById('email');
+const message = document.getElementById('message');
+function storeValues() {
+  localStorage.setItem('f_name', namef.value);
+  localStorage.setItem('email', email.value);
+  localStorage.setItem('message', message.value);
+}
```
Comment (LionRouge1) [inline public/script.js]: - [ ] Kindly create a single JavaScript object with all the data from the entire form and save it in local storage. Do not create one local storage entry per input field.

File: public/script.js
Comment (LionRouge1) [inline public/script.js:1]: - [ ] Kindly make sure the data is saved when the user changes the content of any input field.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @MenendezON, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Followed JavaScript and HTML & CSS best practices :+1: 
- Good commit :heavy_check_mark: 
- All linters passed :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED 💯 

Hi @MenendezON , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Collaborative work 👍 . Commits from both team members
:heavy_check_mark: Use of single JavaScript object to store all input data
:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file and PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**